gateway/src/cluster/impl.rs

```diff
@@ -145,9 +145,14 @@ impl Cluster {
                     .await
                     .map_err(|source| ClusterStartError::RetrievingGatewayInfo { source })?;
 
-                [0, gateway.shards - 1, gateway.shards]
+                [0, gateway.shards - 1, 1, gateway.shards]
             }
-            ShardScheme::Range { from, to, total } => [*from, *to, *total],
+            ShardScheme::Bucket {
+                bucket_id,
+                concurrency,
+                total,
+            } => [*bucket_id, *total, *concurrency, *total],
```

Comment:
It should be documented why the bucket ID is the initial shard ID for future readers, to those that aren't familiar it's not immediately obvious why this is.